CONTRIBUTING.md

```diff
@@ -83,8 +83,7 @@ Before submitting an enhancement suggestion, check if there is already a filed o
 
 First of all, you can validate it with the community, our official channels are:
 
-- [Telegram group](https://t.me/vesperfinance/)
-- [Discord](https://discord.com/invite/Pm8UDaazsK)
+- [Discord](https://discord.gg/hemixyz)
```

Comment:
Not sure if relevant, but one option is to add the [twitter/x account](https://twitter.com/hemi_xyz/) giving the lack of Telegram

Reply:
Do not spend more than 30 seconds on this one. Literally.

Reply:
Added here: https://github.com/hemilabs/pure.finance/pull/2/commits/50613c94ab27cb3cc111f820f6ba587e14cf0d97#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055